src/discord/events/message/registerTeam.ts

```diff
@@ -121,9 +121,10 @@ export const regSubCommands: SubCommands = {
           ],
         },
       });
-      ctx.currentCommand = undefined;
-      ctx.nextStep = undefined;
+      // ctx.currentCommand = undefined;
+      // ctx.nextStep = undefined;
       await ctx.save();
```

Comment:
Investigate whether this method does the same thing as the three lines above; if so, remove this